composer.json

```diff
@@ -15,7 +14,7 @@
         "symfony/translation": "~2.3",
         "pagerfanta/pagerfanta": "dev-master",
         "franmomu/silex-pagerfanta-provider": "dev-master",
-        "doctrine/dbal": "2.2.*",
+        "doctrine/dbal": "v2.5.1",
```

Comment:
The `v` is ignored, it is up to the package to do what it wants. However, I would keep the same style throughout the file.

Do you actually need `>=2.5.1`? It seems `~2.5` would do the trick.